api/src/main/java/org/commonjava/indy/conf/DefaultIndyConfiguration.java

```diff
@@ -239,4 +282,12 @@ public InputStream getDefaultConfig()
         return Thread.currentThread().getContextClassLoader().getResourceAsStream( "default-main.conf" );
     }
 
+    @Override
+    public Properties getSystemProperties()
```

Comment:
`indy.node.id` is also exposed as a system property for later Infinispan interpolation, etc.